### PR TITLE
Update ClusterLogForwarder enhancement proposal to reflect current status.

### PR DESCRIPTION
**Pull request by alanconway**

### CODE REVIEW

**Comment by alanconway** [comment]
@sudhaponnaganti  can I get this approved? Doc changes only.

File: enhancements/cluster-logging/cluster-logging-log-forwarding.md [hunk 1]
```diff
@@ -81,12 +81,12 @@ We will be successful when:
 Provide a declarative `pipeline` that associates a set of named `inputs` with a set of named `outputs`.
 
 The following reserved input names are defined for the initial release:
-* `Application` - Container logs generated by user applications running on the platform, excluding `Infrastructure` containers.
-* `Infrastrucure` - Logs generated by infrastructure components running on the platform and OKD nodes (e.g. journal logs).  "Infrastructure" applications are defined as any pods which run in namespaces: `openshift*`, `kube*`, `default`.
-* `Audit` - Logs generated by the nodes' auditd (/var/log/audit/audit.log), audit logs from the kubeapi-server and the openshift-apiserver. This will not be forwarded by default.
+* `application` - Container logs generated by user applications running on the platform, excluding `infrastructure` containers.
+* `infrastrucure` - Logs generated by infrastructure components running on the platform and OKD nodes (e.g. journal logs).  "infrastructure" applications are defined as any pods which run in namespaces: `openshift*`, `kube*`, `default`.
```
Comment (jcantrill) [inline enhancements/cluster-logging/cluster-logging-log-forwarding.md:85]: spelling

Comment (alanconway) [reply enhancements/cluster-logging/cluster-logging-log-forwarding.md]: fixed now, for real.

**Pull request merged**